vmdb/app/views/shared/dialogs/_dialog_provision.html.haml

```diff
@@ -15,7 +15,7 @@
       - wf.dialog.dialog_tabs.each do |tab|
         = render :partial => "shared/dialogs/dialog_tab", 
                  :locals => {:wf => wf, :tab => tab}
-  - unless @edit[:explorer]
+  - if @edit && !@edit[:explorer]
```

Comment:
why is @edit nil in some cases?  Seems like lots of places would have to check for nil @edit